### PR TITLE
Add new case note button

### PR DESCRIPTION
**Pull request by andrewburrowsmoj**
## What does this pull request do?

Provides a button to allow the user to start the draft journey for creating a new case note.

## What is the intent behind these changes?

Allow user to access create a case note journey.

## Screenshot

image

### CODE REVIEW

File: server/routes/caseNotes/confirmation/newCaseNoteConfirmationPresenter.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { SummaryListItem } from '../../../utils/summaryList'
+import PresenterUtils from '../../../utils/presenterUtils'
+import utils from '../../../utils/utils'
+import SentReferral from '../../../models/sentReferral'
+import Intervention from '../../../models/intervention'
+
+export default class NewCaseNoteConfirmationPresenter {
```
Comment (Gweaton) [inline server/routes/caseNotes/confirmation/newCaseNoteConfirmationPresenter.ts:7]: As mentioned on an earlier PR, do we want to rename this to `AddCaseNote...` for consistency?

File: server/routes/caseNotes/confirmation/newCaseNoteConfirmationView.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import ViewUtils from '../../../utils/viewUtils'
+import NewCaseNoteConfirmationPresenter from './newCaseNoteConfirmationPresenter'
+
+export default class NewCaseNoteConfirmationView {
```
Comment (Gweaton) [inline server/routes/caseNotes/confirmation/newCaseNoteConfirmationView.ts:4]: And here too (sorry!)

**Review by Gweaton — APPROVED** [review]
Just a couple of naming things, as is my trademark

File: server/routes/caseNotes/confirmation/newCaseNoteConfirmationPresenter.ts (hunk 1 above)
Comment (andrewburrowsmoj) [reply server/routes/caseNotes/confirmation/newCaseNoteConfirmationPresenter.ts:7]: hmmm, I renamed it 

ah looks like these were included accidently when rebasing (the new renamed classes exist and these old ones aren't being used) I'll update the commit to remove them

**Pull request merged**